decentraland/unity-renderer
# feat: skybox tool new UI

## What does this PR change?

- New UI for Skybox tool
- For transitioning variables now can add points in-between


## How to test the changes?



1. - Open in Unity and goto Tools -> Skybox Editor
2. Can check build : https://play.decentraland.zone/?renderer=https://glob-dland-s3-unity-renderer.s3.amazonaws.com/branch/feat/skybox-tool-new-ui

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
